tests/functional-tests/README.md

```diff
@@ -16,62 +13,60 @@ This project contains an automation framework to exercise the IBM Spectrum Scale
-        python3.7 -m pip install jsmin
-  ```
-       
+``` 
+python3.7 -m pip install kubernetes
+python3.7 -m pip install pytest
+python3.7 -m pip install pytest-html
+python3.7 -m pip install jsmin
+```
 
 ### How to run IBM Spectrum Scale CSI test automation
-- Clone the source code from [git](https://github.com/IBM/ibm-spectrum-scale-csi/) repository.
-
-- Configure parameters in [tests/functional-tests/config.json](https://github.com/IBM/ibm-spectrum-scale-csi/blob/dev/tests/functional-tests/config.json) file. 
-
-- Set node labels for "attacherNodeSelector","provisionerNodeSelector","pluginNodeSelector" in config.json file
-
-
-- cluster-config parameter is mandatory to pass the SpectrumScale & CSI Operator/Driver images configuration as input.
 
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json
-                   
-- If kubeconfig file is at not at ~/.kube/config location, pass the correct location with --kubeconfig (optional)               
-                
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json --kubeconfig <kubeconfig_file_path>
+- Configure parameters in [config.json](./tests/functional-test/config.json) file.
+
+- Define kubernetes/Openshift Spectrum Scale node labels for `attacherNodeSelector`,`provisionerNodeSelector`,`pluginNodeSelector` in config.json file
+
+- `--clusterconfig` parameter is mandatory to pass IBM Spectrum Scale API Credentials & CSI Operator/Driver image specific configuration as input.
+```
+For example :
+  pytest  driver_test.py --clusterconfig config.json
+```                   
+- If kubeconfig file is at not at `~/.kube/config` location, pass the correct location with `--kubeconfig` (optional)
+```
+For example :
+  pytest  driver_test.py --clusterconfig config.json --kubeconfig <kubeconfig_file_path>
+```
+- For generating the report in html format, pass the `--html` (optional)
+```
+For example :
+  pytest  driver_test.py --clusterconfig config.json --html report.html
+```
 
-- For generating the report in html format, pass the --html (optional)               
-                
-                For example :
-                   pytest  driver_test.py --clusterconfig config.json --html report.html
-                   
-                
 ### Run driver tests using driver_test.py as shown below -
-       
-                cd csi-automation/scale_operator
+```
+cd ./tests/functional-tests/
 
-                #Run all testcases in driver testsuite using:
-                pytest  driver_test.py --clusterconfig config.json
+#Run all testcases in driver testsuite using:
+pytest  driver_test.py --clusterconfig config.json
 
-                #Run any specific testcase in testsuite using:
-                pytest  driver_test.py::<test_name> --clusterconfig config.json
+#Run any specific testcase in testsuite using:
+pytest  driver_test.py::<test_name> --clusterconfig config.json
```

Comment:
How do users find out what are the lists of valid `<test_name>`, this might be useful.  

Are there any examples or pre-defined testing "bundles"?

Reply:
Good catch, though we have mentioned the example but yes we have not created any specific documentation for list of testcases. We will try to make the comprehensive list of test cases in next check-in.

Reply:
@Jainbrt just to be clear, I'm not asking for a comprehensive list and/or manual documentation around test cases.   I think that's just going to add to creating document that would go out of sync, and become un-maintainable.  

I think if we could self document the testcases where then we could generate docs, that is 👍 ..  or a way to dynamically show the tests.. and their purpose in a snip... 

just for example:
```
./test..  --list  # shows short name of list taken from some self documenting heading
./test --list -v # shows a short snip or description, taken from some self documenting heading 
```

Reply:
@whowutwut There is a option in pytest which helps you to list the available test
 `pytest --collect-only`